packages/shared/src/components/sidebar/Sidebar.tsx

```diff
@@ -72,6 +81,27 @@ export default function Sidebar({
   });
   const newSquadButtonVisible =
     sidebarRendered && hasSquadAccess && !squads?.length;
+  const newSquadTooltipOffset: [number, number] = sidebarExpanded
+    ? [0, 1.5 * 16]
+    : [0, 1 * 16];
+  const newSquadTooltipTutorial = useTutorial({
+    key: TutorialKey.SEEN_NEW_SQUAD_TOOLTIP_KEY,
+  });
+
+  useEffect(() => {
+    if (!newSquadTooltipTutorial.isActive) {
+      return undefined;
+    }
+
+    const timeout = setTimeout(() => {
+      newSquadTooltipTutorial.complete();
+    }, 4 * 1000);
+
+    return () => {
+      clearTimeout(timeout);
+    };
+  }, [newSquadTooltipTutorial.isActive]);
```

Comment:
We have some hooks I think that deal with timers already would suggest we stick to those.

Either `useTimedAnimation` or `useTimer` if they don't fit let's see how we can abstract into a better hook maybe?